wheel_separation: 0.09
wheel_radius: 0.0325
base_name: "base_wheel"
wheel_name: "left_wheel"
right_wheel: false